website/pages/docs/nia/tasks.mdx

```diff
@@ -88,7 +88,7 @@ For complete information on the event structure, see [events in our API document
 
 ### Task Status
 
-Each time a task runs to update network infrastructure, event data is stored for that run. Currently, only the 5 most recent events are stored for each task. The stored events are used to determine the task status. For example, if a task has 5 events stored and 3 are success and 2 are not, then the task's health status is "degraded".
+Each time a task runs to update network infrastructure, event data is stored for that run. Currently, only the 5 most recent events are stored for each task. The stored events are used to determine the task status. For example, if a task has 5 events stored and the most recent is not successful but all previous ones are, then the task's health status is "degraded".
```

Comment:
This reads like it still needs to be updated, mix of old and new logic.

Reply:
Ack, a 'degraded' made it through. Thanks! What do you think about below?

> Each time a task runs to update network infrastructure, event data is stored for that run. Currently, only the 5 most recent events are stored for each task, and these stored events are used to determine task status. For example, if the most recent stored event is not successful but the others are, then the task's health status is "errored"

Reply:
I would suggest to remove the "Currently, only the 5" and just go with a statement that "5 most recent events are stored for each task, ..."

The new changes look good 👍

Reply:
Note: seeing another 'degraded' a little lower down line 97